src/quickdocumentdialog.cpp

```diff
@@ -491,46 +520,81 @@ void QuickDocumentDialog::addUserPaper()
 	}
 }
 
-void QuickDocumentDialog::addUserInputEncoding()
+void QuickDocumentDialog::addUserFontEncoding()
 {
 	QString newoption;
 	UniversalInputDialog dialog;
 	dialog.addVariable(&newoption, tr("New:"));
 	if (dialog.exec() && !newoption.isEmpty()) {
-		otherInputEncodingList.append(newoption);
+		otherFontEncodingList.append(newoption);
 		Init();
 	}
 }
 
-void QuickDocumentDialog::addUserFontEncoding()
+void QuickDocumentDialog::addBabelOption()
 {
 	QString newoption;
 	UniversalInputDialog dialog;
 	dialog.addVariable(&newoption, tr("New:"));
 	if (dialog.exec() && !newoption.isEmpty()) {
-		otherFontEncodingList.append(newoption);
+		otherBabelOptionsList.append(newoption);
 		Init();
 	}
 }
 
-void QuickDocumentDialog::addBabelOption()
+void QuickDocumentDialog::addUserOptions()
 {
 	QString newoption;
 	UniversalInputDialog dialog;
 	dialog.addVariable(&newoption, tr("New:"));
 	if (dialog.exec() && !newoption.isEmpty()) {
-		otherBabelOptionsList.append(newoption);
+		otherOptionsList.append(newoption);
 		Init();
 	}
 }
 
-void QuickDocumentDialog::addUserOptions()
+void QuickDocumentDialog::addUserPackages()
 {
```

Comment:
this dialog is a bit underwhelming, especially as txs has a list of all available packages.
But maybe a task for a future PR.

Reply:
I know